src/keboola/component/base.py

```diff
@@ -110,3 +136,72 @@ def execute_action(self):
         except AttributeError as e:
             raise AttributeError(f"The defined action {action} is not implemented!") from e
         return action_method()
+
+    def create_out_table_definition_from_schema_name(self, schema_name: str, is_sliced: bool = False,
```

Comment:
Wouldn't `create_out_table_definition_from_schema` name be enough? BTW reminds me of this https://keboola.atlassian.net/browse/KINTER-98 :D

Reply:
Agreed, added in next commit